unstructured/partition/md.py

```diff
@@ -86,7 +86,7 @@ def partition_md(
 
         text = response.text
 
-    html = markdown.markdown(text)
+    html = markdown.markdown(text, extensions=['tables'])
```

Comment:
does this require `markdown.extensions` package?

Reply:
apparently not